packages/model-viewer/src/features/loading.ts

```diff
@@ -432,6 +432,7 @@ export const LoadingMixin = <T extends Constructor<ModelViewerElementBase>>(
             // screen readers
             defaultPosterElement.setAttribute('aria-hidden', 'true');
             defaultPosterElement.tabIndex = -1;
+            this.dispatchEvent(new CustomEvent('poster-dismissed'));
```

Comment:
Have you verified this fixes most of the flakey renders? Which ones are still experiencing flakes, and roughly how often (%)?

Reply:
i run the fidelity test 3 times so far, all 100% passed. I will keep running it and then tell you the result